docs/src/examples/components/Button/States/ButtonExampleLoading.shorthand.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { Button, Flex } from '@stardust-ui/react'
+import { useBooleanKnob } from '@stardust-ui/docs-components'
+
+const ButtonExampleLoading = () => {
+  const [loading] = useBooleanKnob({ name: 'loading', initialValue: true })
+
+  return (
+    <Flex column gap="gap.smaller">
+      <Flex gap="gap.smaller">
+        <Button loading={loading} content="Default" />
+        <Button loading={loading} content="Primary" primary />
+        <Button loading={loading} content="Secondary" secondary />
+        <Button loading={loading} icon="book" content="Click me" iconPosition="before" primary />
+        <Button loading={loading} circular icon="coffee" />
+      </Flex>
+      <Button loading={loading} fluid content="Fluid" />
+    </Flex>
+  )
+}
+
+export default ButtonExampleLoading
```

Comment:
Do we need to have all of these in this example? Can we have just `<Button content="Processing..." loading />` and move all others to hidden visual tests section? What do you think?

Reply:
Make sense!